tower-service/Cargo.toml

```diff
@@ -17,6 +17,7 @@ description = """
 Trait representing an asynchronous, request / response based, client or server.
 """
 categories = ["asynchronous", "network-programming"]
+edition = "2018"
```

Comment:
`tower-service` cannot be switched to 2018. Doing so will prevent crates that are unable to update their minimum rust version to implement `Service`.

Reply:
I can bump the version number to 0.3, signifying a breaking change, or bring it down to the 2015 edition. Which would you prefer?

Reply:
IMHO this is not worth a breaking change right now.

Reply:
Understood, I'll bump it down to 2015.